### ISSUE TITLE
Magnet search result does not return back to search list

### ISSUE THREAD

**Issue by Invictaz** [LivingWithHippos/unchained-android#116]

**Comment by LivingWithHippos** [comment]
Yes I pass the links from the search screen to the new download screen so that's technically correct. Will see If I can edit the backstack somehow

**Comment by Invictaz** [comment]
Not after every search result is a new download needed. Sometimes a search list or other screen. 

In my understanding pressing "back" on the phone button always displays the previous screen.

**Comment by LivingWithHippos** [comment]
If you press on a magnet or torrent link after the search it gets passed to the new download fragment and downloaded this is why pressing back you get that fragment

**Comment by Invictaz** [comment]
Would be better if it would go back go search results (if there were any)
